test/cache_test.rb

```diff
@@ -330,7 +374,7 @@ def test_uses_file_digest_in_cache_key
     end
 
     def test_cache_digest_definition
-      assert_equal(::Model::FILE_DIGEST, @post_serializer.class._cache_digest)
+      assert_equal(FILE_DIGEST, @post_serializer.class._cache_digest)
```

Comment:
not sure how I feel about this assertion.. maybe this should just be ` Digest::MD5.hexdigest(File.open(__FILE__).read)`